### ISSUE TITLE
Release 2022.11.0

### ISSUE THREAD

**Issue by jrbourbeau** [dask/community#287]
|  |  |
| --- | --- |
| Release version | 2022.11.0 |
| Planned release date | 2022-11-15 |
| Status | On-track |
| Release manager | @jrbourbeau |

## Best effort

_Try to close before the release but will not block the release_

- [x] https://github.com/dask/distributed/pull/5648
- [x] https://github.com/dask/dask/pull/9659
- [x] https://github.com/dask/dask/pull/9642
- [x] https://github.com/dask/dask/pull/9570
- [ ] ...

## Blocker

_Issues that would cause us to block and postpone the release if not fixed_

- [ ] ...
- [ ] ...

## Comments

cc @fjetter @quasiben @jakirkham @rjzamora

**Comment by jrbourbeau** [comment]
Adding https://github.com/dask/dask/pull/9570 to the "Best effort" list

**Comment by jakirkham** [comment]
JFYI RAPIDS (US) will be on holiday tomorrow

**Comment by jrbourbeau** [comment]
Thanks for the heads up @jakirkham. US-based folks at Coiled will be off too. In light of that, I'll propose we bump the release to next Monday so more folks will be around to help if any follow-up is needed for the release (this isn't usually needed, but does happen rarely). I also don't think there's anything so pressing sitting in `main` that can't wait until next Monday.

**Comment by jrbourbeau** [comment]
I wanted to surface some thoughts on https://github.com/dask/distributed/pull/5648 here https://github.com/dask/distributed/pull/5648#issuecomment-1314071384. I'm proposing we include that PR in the release today. With the changes in that PR, user code won't break when `bokeh=3` is installed. There are a couple of plots with CSS/sizing issues (it's still not clear if this is an issue with `distributed` or `bokeh`), but I'd rather have a couple of plots be off than have users running into `ImportError`s. If folks have thoughts on this topic, I'd appreciate whatever feedback you have.

**Comment by jakirkham** [comment]
Would those changes still work with Bokeh 2 or does Bokeh 3 become the new minimum requirement?

**Comment by jrbourbeau** [comment]
Yeah, everything is backwards compatible with `bokeh=2`. The only dependency change we're making is removing the `bokeh<3` constraint

**Comment by jrbourbeau** [comment]
Status update: It looks like there's agreement around including https://github.com/dask/distributed/pull/5648 in the release. The `bokeh=3.0.2` release fixes some CI failures we seeing in https://github.com/dask/distributed/pull/5648. `bokeh=3.0.2` is on PyPI but isn't yet on `conda-forge` (but is close https://github.com/conda-forge/bokeh-feedstock/pull/85). I'll propose we push the release back one more day to give `bokeh=3.0.2` time to land on conda-forge. We can then merge https://github.com/dask/distributed/pull/5648 / https://github.com/dask/dask/pull/9659 and release.

**Comment by jrbourbeau** [comment]
Okay, all outstanding issues for this release have been addressed. I'm going to start pushing out the release in the next hour

**Comment by jrbourbeau** [comment]
2022.11.0 is on PyPI and conda-forge PRs are open:

- [x] https://github.com/conda-forge/dask-core-feedstock/pull/153
- [x] https://github.com/conda-forge/distributed-feedstock/pull/233
- [x] https://github.com/conda-forge/dask-feedstock/pull/207

**Comment by jakirkham** [comment]
Conda & PyPI packages are up.

Currently waiting on the Docker image update ( https://github.com/dask/dask-docker/pull/262 ).

**Comment by charlesbluca** [comment]
Docker images are updated with dask/dask-docker#263

**Comment by jrbourbeau** [comment]
Thanks for you work on getting that across the finish line @charlesbluca!
